login-workflow/CHANGELOG.md

```diff
@@ -5,11 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v4.0.4 (Unreleased)
+## v5.0.0 (Unreleased)
 
 ### Added
 
--   Error Manager for ChangePasswordDialog ([#612](https://github.com/etn-ccis/blui-react-workflows/issues/612))
+-   Reusable okta login screen.
+-   Error Manager for ChangePasswordDialog ([#612](https://github.com/etn-ccis/blui-react-workflows/issues/612)).
+-   Reload EULA should not be tied to checkbox status ([#549](https://github.com/etn-ccis/blui-react-workflows/issues/549)).
+-   Eliminate `WorkflowCardFinishState` and introduce `EmptyStateProps` in the success screen. ([#552](https://github.com/etn-ccis/blui-react-workflows/issues/552)).
+-   Updated password validation callaback function.([#560]https://github.com/etn-ccis/blui-react-workflows/issues/560).
+-   SX Style overrides on full screens([#599](https://github.com/etn-ccis/blui-react-workflows/issues/599)).
+-   Added BLUI class names for ChangePasswordDialog ([#600](https://github.com/etn-ccis/blui-react-workflows/issues/600)).
+
```

Comment:
These should not all be under the 'Added' category. Some are 'Changed' 'Removed' or 'Fixed'. Refer to https://keepachangelog.com/en/1.1.0/

Reply:
> These should not all be under the 'Added' category. Some are 'Changed' 'Removed' or 'Fixed'. Refer to https://keepachangelog.com/en/1.1.0/

Oh yes! I missed this, sorry.